doc/man/8/rbdmap.rst

```diff
@@ -18,14 +18,78 @@ Description
 
 **rbdmap** is a shell script that can be run manually by the system
 administrator at any time, or automatically at boot time by the init system
-(sysvinit, upstart, systemd). The script looks for an environment variable
-``RBDMAPFILE``, which defaults to ``/etc/ceph/rbdmap``. This file is
-expected to contain a list of RBD images and, possibly, parameters to be
-passed to the underlying ``rbd`` command. The syntax of
-``/etc/ceph/rbdmap`` is described in the comments at the top of that file.
-
-The script mounts devices after mapping, and unmounts them before
-unmapping.
+(sysvinit, upstart, systemd).
+
+The script takes a single argument, which can be either "map" or "unmap".
+In either case, the script parses a configuration file (defaults to ``/etc/ceph/rbdmap``,
+but can be overrided via an  environment variable ``RBDMAPFILE``). Each line
+of the configuration file corresponds to an RBD image which is to be mapped, or
+unmapped.
+
+The configuration file format is::
+
+    IMAGESPEC RBDOPTS
+
+where ``IMAGESPEC`` should be specified as ``POOLNAME/IMAGENAME`` (the pool
+name, a forward slash, and the image name), and ``RBDOPTS`` is a s
+
+Images are mapped as ``/dev/rbd/$IMAGESPEC``, where ``$IMAGESPEC`` is the device name
```

Comment:
@dillaman If the udev rules are doing the mapping, do we need `rbdmap.service` at all? I'm thinking that if the udev rules fire before `remotefs.target` runs, then an RBD image could be treated just like an NFS mount in /etc/fstab ?